phpstan-baseline.neon

```diff
@@ -45,6 +45,11 @@ parameters:
 			count: 1
 			path: lib/Doctrine/ORM/Cache/Persister/Collection/AbstractCollectionPersister.php
 
+		-
+			message: "#^Else branch is unreachable because ternary operator condition is always true\\.$#"
+			count: 1
+			path: lib/Doctrine/ORM/Cache/Persister/Collection/AbstractCollectionPersister.php
+
```

Comment:
Where does this happen?

Reply:
In the method `storeCollectionCache()`, PHPStan believes `$elements instanceof Collection` to be always true, but the parameter `$elements` is declared as `Collection|mixed[]`. This seems to be a false positive.